abi/order.go

```diff
@@ -0,0 +1,150 @@
+package abi
+
+import (
+	"context"
+	"math/big"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/orbs-network/order-book/utils/logger"
+	"github.com/orbs-network/order-book/utils/logger/logctx"
+)
+
+// function call ABI
+const executeBatchJson = `[{
+	"inputs": [
+		{
+			"components": [
+				{
+					"internalType": "bytes",
+					"name": "order",
+					"type": "bytes"
+				},
+				{
+					"internalType": "bytes",
+					"name": "sig",
+					"type": "bytes"
+				}
+			],
+			"name": "orders",
+			"type": "tuple[]"
+		}
+	],
+	"name": "executeBatch",
+	"outputs": [],
+	"stateMutability": "payable",
+	"type": "function"
+}]`
+
+type Info struct {
```

Comment:
I have already defined these types in `models/abi_order.go`. Can we move what is there into this `abi` package so we don't duplicate?